machines/openIdBle/request.ts

```diff
@@ -409,7 +424,33 @@ export const requestMachine =
                 },
               },
             },
-            navigatingToHome: {},
+            navigatingToHome: {
+              invoke: {
+                src: 'disconnect',
+              },
+            },
+            savingFailed: {
+              initial: 'idle',
+              entry: ['setReceiveLogTypeDiscarded', 'logReceived'],
+              invoke: {
+                src: 'sendVcResponse',
+                data: {
+                  status: 'REJECTED',
+                },
+              },
+              states: {
+                idle: {},
+                viewingVc: {},
+              },
+              on: {
+                DISMISS: {
+                  target: '.viewingVc',
+                },
+                GO_TO_RECEIVED_VC_TAB: {
```

Comment:
On saving failed, do we need to handle GO_TO_RECEIVED_VC_TAB event?

Reply:
Yes we need. Even if failed, We are showing VC to the user with a button to go to received vc page

Reply:
Okay